src/main/java/org/eclipse/microprofile/starter/addon/microprofile/servers/model/JDKSelector.java

```diff
@@ -44,6 +44,7 @@ public void init() {
         fillJavaSEVersion(data, SupportedServer.THORNTAIL_V2, JavaSEVersion.SE11, MicroProfileVersion.MP22, null);  // Supported from MP 2.2
 
         fillJavaSEVersion(data, SupportedServer.QUARKUS, JavaSEVersion.SE8, null, null);  // Supported for all MPVersions
+        fillJavaSEVersion(data, SupportedServer.QUARKUS, JavaSEVersion.SE11,  MicroProfileVersion.MP32, null);  // Supported for all MPVersions
```

Comment:
Comment says all versions but code configures it from MP3.2 and higher.

Reply:
THX. Fixed the comment.